castle/cms/testing.py

```diff
@@ -7,6 +7,10 @@
 from plone.app.testing import MOCK_MAILHOST_FIXTURE
 from plone.app.testing import PLONE_FIXTURE
 from plone.app.testing import PloneSandboxLayer
+from plone.app.testing import login
```

Comment:
It looks to me like you added these imports but didn't need them because you added no code that use them. If that's the case, you should undo/revert the change to this file. We want to change only what is needed.